GameServerLib/Scripting/CSharp/CSharpScriptEngine.cs

```diff
@@ -53,50 +72,58 @@ public bool Load(List<string> scriptLocations)
             var assemblyName = Path.GetRandomFileName();
 
             var references = new List<MetadataReference>();
-            foreach (var a in AppDomain.CurrentDomain.GetAssemblies())
-                if (!a.IsDynamic && !a.Location.Equals(""))
-                    references.Add(MetadataReference.CreateFromFile(a.Location));
+            foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
```

Comment:
Have you tested performance on both for and foreach?

Reply:
I don't think performance is necessary here. This should only be called once on game start or when manually reloading all scripts.
For is said to be 2-5x faster but sacrifices in readability. There won't be a perceivable difference here in performance.